Tests/FluentAssertions.Net45.Specs/Net45.Specs.csproj

```diff
@@ -84,6 +84,9 @@
     <Otherwise />
   </Choose>
   <ItemGroup>
+    <Compile Include="..\FluentAssertions.Net40.Specs\XmlElementAssertionSpecs.cs">
```

Comment:
You're right. The purpose of that shared project has gone beyond my initial goal by all the refactoring.